### PR TITLE
Move from Pods to Jobs

### PR DESCRIPTION
**Pull request by fridex**
Depends-On: https://github.com/thoth-station/common/pull/138

### CODE REVIEW

**Comment by goern** [comment]
recheck

File: thoth_user_api/api_v1.py [hunk 1]
```diff
@@ -62,12 +62,12 @@ def get_analyze(analysis_id: str):
 
 def get_analyze_log(analysis_id: str):
     """Get image analysis log."""
-    return _get_pod_log(locals(), 'package-extract-', Configuration.THOTH_MIDDLETIER_NAMESPACE)
+    return _get_job_log(locals(), 'package-extract-', Configuration.THOTH_MIDDLETIER_NAMESPACE)
```
Comment (goern) [inline thoth_user_api/api_v1.py]: Should https://github.com/thoth-station/common/blob/master/thoth/common/openshift.py#L299 be called??

Comment (goern) [reply thoth_user_api/api_v1.py]: no... now I see :)

Comment (fridex) [reply thoth_user_api/api_v1.py]: It is, in the ``_get_job_log`` wrapper.

**Pull request merged**